<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddRoomWindow</class>
 <widget class="QMainWindow" name="AddRoomWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>403</width>
    <height>614</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>403</width>
    <height>614</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>403</width>
    <height>614</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Zapchat</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../res/img/Zapchat.png</normaloff>../res/img/Zapchat.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <widget class="QPushButton" name="goto_chat_button">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>540</y>
      <width>150</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	background-color: #003bbd;
	color: rgb(255, 255, 255);
	font-weight: bold;
	border-style: solid;
	border-width: 1px;
	border-radius: 5px;
	padding: 10px;
	font: 700 9pt &quot;Exo 2&quot;;


}
</string>
    </property>
    <property name="text">
     <string>Find Room</string>
    </property>
   </widget>
   <widget class="QLabel" name="profile_pic">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid;
border-color: rgb(0, 0, 0);</string>
    </property>
    <property name="pixmap">
     <pixmap>../res/img/temp_profile_pic.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="profile_pic_button">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>170</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../res/icon/addPhoto_icon.png</normaloff>../res/icon/addPhoto_icon.png</iconset>
    </property>
   </widget>
   <widget class="QGroupBox" name="info_le_gb">
    <property name="geometry">
     <rect>
      <x>29</x>
      <y>350</y>
      <width>341</width>
      <height>150</height>
     </rect>
    </property>
    <property name="title">
     <string>Info - optional</string>
    </property>
    <widget class="QTextEdit" name="info_te">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>321</width>
       <height>120</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-style:solid;
border-width:5px;
border-color: rgb(255, 255, 255);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <property name="placeholderText">
      <string>Enter Your Info</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>361</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget
{
	color: #263238;
  	font: 900 14pt &quot;Exo 2&quot;;
}</string>
    </property>
    <property name="text">
     <string>Room Info</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>110</y>
      <width>211</width>
      <height>60</height>
     </rect>
    </property>
    <property name="title">
     <string>ID</string>
    </property>
    <widget class="QLineEdit" name="name_le_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>191</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-left-color: #fff;
border-left-style: solid;
border-left-width: 5px;
text-indent: 3px;
padding-left:3px;
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="maxLength">
      <number>20</number>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
     <property name="placeholderText">
      <string>Enter Your Name</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="name_le_gb">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>230</y>
      <width>241</width>
      <height>60</height>
     </rect>
    </property>
    <property name="title">
     <string>Name</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <widget class="QLineEdit" name="name_le">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>221</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">padding-left:5px;
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="maxLength">
      <number>20</number>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
     <property name="placeholderText">
      <string>Enter Your Name</string>
     </property>
    </widget>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>37</x>
      <y>310</y>
      <width>323</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>250</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Group</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../res/icon/group_room_icon.png</normaloff>../res/icon/group_room_icon.png</iconset>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Channel</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../res/icon/channel_room_icon.png</normaloff>../res/icon/channel_room_icon.png</iconset>
     </property>
    </item>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
